Remove license text from index.json of licenseDB

AyanSinhaMahapatra opened the pull request:
* Removes license text from index.json and index.yaml
* Adds license text to respective json and yaml files.
* Adds spdx license list version https://github.com/nexB/scancode-licensedb/issues/11

Signed-off-by: Ayan Sinha Mahapatra <ayansmahapatra@gmail.com>

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

AyanSinhaMahapatra committed:
Remove license text from index.json of licenseDB

* Removes license text from index.json and index.yaml
* Adds license text to respective json and yaml files.

Signed-off-by: Ayan Sinha Mahapatra <ayansmahapatra@gmail.com>

pombredanne commented on tests/licensedcode/data/license_db/license_dump/index.html:
```diff
@@ -371,6 +371,7 @@
     </p>
     <p>Designed and built by <a href="https://www.nexb.com/" target="_blank">nexB</a>. Licensed under the <a href="cc-by-4.0.html">Creative Commons Attribution License 4.0 (CC-BY-4.0)</a>.</p>
     <p>Generated with <a href="https://github.com/nexB/scancode-toolkit" target="_blank">ScanCode toolkit</a> 32.0.0b1 on Dec 22, 2022.</p>
+    <p>This is updated daily automatically with latest updates from the develop branch of scancode-toolkit, if any.</p>
```
we should IMHO tell which exact commit and link to it.
The `toolkit</a> 32.0.0b1 on Dec 22, 2022.</p>` thing is  misleading.

pombredanne committed:
Improve .VERSION for git export-subst

We now have a correct abbreviated commit, a commit date and a git
decribe. For example:

refs=HEAD -> fix-license-dump
commit=5ccc92e69cffb503e9bedc7fce5a1dbb0fd851da
abbrev_commit=5ccc92e69c
committer_date=2023-01-16
git_describe=v31.2.3-328-g5ccc92e69c

Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

pombredanne committed:
Improve documented versions in the license dump

* Add a better version in scancode-config, using git describe
* Add SPDX license list version to the HTML footer

Reference: https://github.com/nexB/scancode-toolkit/issues/3200
Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

pombredanne reviewed (approved):
LGTM! Thanks!
